API/ExecutorInterface.php

```diff
@@ -27,4 +27,13 @@ public function supportedTypes();
      * @throws \Exception
      */
     public function execute(MigrationStep $step);
+
+    /**
+     * Generate a update template
+     *
+     * @param string $identifier
+     * @return string Yaml dump
+     * @throws \Exception
+     */
+    public function generateTemplate($identifier);
```

Comment:
I think that we might rename this method generateMigration instead of generateTemplate ?
Also maybe we could add a 2nd parameter which would switch between generating create and update migrations?